src/cpu/cp0/reg_config.rs

```diff
@@ -18,6 +19,12 @@ impl From<u32> for RegConfig {
             endianness: value.into(),
 
             cu: (value & (1 << 3)) != 0,
+            kseg0_cache_enable_bits: [
+                (value & (1 << 0)) != 0,
+                (value & (1 << 1)) != 0,
+                (value & (1 << 2)) != 0,
+            ],
+            // TODO: Extract to method?
```

Comment:
Thanks for adding this comment as well; I had the same thought since we shouldn't really be storing the same information twice.